modules/@ergonode/product-templates/src/components/TemplateDesigner/ProductDesigner/LayoutElement.vue

```diff
@@ -355,9 +342,12 @@ export default {
             );
 
             if (height <= this.maxHeight && height >= this.minHeight) {
-                this.newHeight = row - this.element.row + 1;
+                const row = Math.ceil(height / (this.minHeight + (this.gap * 2)));
```

Comment:
looks like line 309, maybe you should create a helper function with dimention/minDimention parameters?

Reply:
Not exactly the same, I would need to create function which would be total abstraction. Same would be for next line:
`const actualRow = this.element.row + row - 1;`
and
`const actualColumn = this.element.column + column - 1;`

Where we can pass two parameters and have function, but what do we gain? To keep readability instead of creating abstract function, I've created two other functions which would tell by name what they will return